core/testing/src/main/java/com/merxury/blocker/core/testing/testing/data/ComponentInfoTestData.kt

```diff
@@ -30,3 +30,20 @@ val componentInfoTestData: SnapshotStateList<ComponentItem> = mutableStateListOf
         type = RECEIVER,
     ),
 )
+
+val componentInfoTestData2: SnapshotStateList<ComponentItem> = mutableStateListOf(
```

Comment:
create test component data for activities, receivers, services and providers?

Reply:
yes